bazel/foreign_cc/nghttp2.patch

```diff
@@ -8,7 +8,7 @@ index 35c77d1d..47bd63f5 100644
    # Set it to "int" to match the behavior of AC_TYPE_SSIZE_T (autotools).
 -  set(ssize_t int)
 +  if(WIN32 AND CMAKE_SIZEOF_VOID_P EQUAL 8)
-+    set(ssize_t int64_t)
++    set(ssize_t ptrdiff_t)
```

Comment:
Can you upstream this? Patches aren't sustainable for us long term.

Reply:
This will not be accepted upstream until a version major bump (note it is our patch - of envoy's present patch.) PR and discussion item here; https://github.com/nghttp2/nghttp2/pull/1395